Flow.Launcher.Core/Plugin/PluginAssemblyLoader.cs

```diff
@@ -11,17 +11,22 @@ internal class PluginAssemblyLoader : AssemblyLoadContext
     {
         private readonly AssemblyDependencyResolver dependencyResolver;
 
-        private readonly AssemblyDependencyResolver referencedPluginPackageDependencyResolver;
+        private static readonly AssemblyDependencyResolver referencedPluginPackageDependencyResolver;
 
         private readonly AssemblyName assemblyName;
 
+        static PluginAssemblyLoader()
+        {
+            referencedPluginPackageDependencyResolver =
+                new AssemblyDependencyResolver(Path.Combine(Constant.ProgramDirectory, "Flow.Launcher.dll"));
+        }
+
         internal PluginAssemblyLoader(string assemblyFilePath)
         {
             dependencyResolver = new AssemblyDependencyResolver(assemblyFilePath);
             assemblyName = new AssemblyName(Path.GetFileNameWithoutExtension(assemblyFilePath));
 
-            referencedPluginPackageDependencyResolver =
-                new AssemblyDependencyResolver(Path.Combine(Constant.ProgramDirectory, "Flow.Launcher.Plugin.dll"));
+
```

Comment:
remove this extra space please while you are at it :P

Reply:
Will do🤣